src/styles/blocks/grid.scss

```
@@ -0,0 +1,32 @@
+.grid {
+  display: grid;
+  grid-gap: 46px 48px;
+  grid-template-columns: repeat(4, $card-width);
+  grid-template-rows: auto;
+  justify-content: center;
+  padding: 50px 0 73px;
+}
+
+
+@media  (max-width: 488px) {
+  .grid {
+    grid-template-columns: repeat(1, $card-width);
+  }
+}
```

```suggestion
.grid {
  display: grid;
  grid-gap: 46px 48px;
  grid-template-columns: repeat(4, $card-width);
  grid-template-rows: auto;
  justify-content: center;
  padding: 50px 0 73px;
  
  @media  (max-width: 488px) {
    grid-template-columns: repeat(1, $card-width);
   }
}



```
Use SASS nesting here